projects/storefrontapp-e2e-cypress/cypress/helpers/b2b/my-company/models/my-company.config.ts

```diff
@@ -125,6 +125,21 @@ export interface MyCompanyConfig {
    */
   createConfig?: MyCompanyConfig;
 
+  /**
+   * SUBCATEGORY: Config for updating entities from submenu
+   */
```

Comment:
Should we add dot here after sentence? Or remove all everywhere to keep it consistent?